debian/postinst

```diff
@@ -12,7 +12,7 @@ case "$1" in
     configure)
 
         # Create custom sudoers file
-        echo "Defaults env_keep += KLOG_FORCE_FLUSH" > $TMP_FILE
+        echo 'Defaults env_keep += "KLOG_FORCE_FLUSH NO_AT_BRIDGE"' > $TMP_FILE
```

Comment:
@Ealdwulf are these quotes okay? Just to double check, cause it's different from what we used before